Resources/Prototypes/Entities/Markers/Spawners/ghost_roles.yml

```diff
@@ -0,0 +1,16 @@
+- type: entity
+  id: SpawnPointGhostRatKing
+  name: ghost role spawn point
+  suffix: rat king
+  parent: MarkerBase
+  components:
+  - type: GhostRoleMobSpawner
```

Comment:
Pretty sure you need `deleteOnSpawn: true` otherwise it'll spawn infinite.

Reply:
It defaults to true

Reply:
![image](https://user-images.githubusercontent.com/60792108/177419778-47a48686-9ea1-46c6-b07a-e9bfe175b4ee.png)